src/js/components/Notification/index.d.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export type StatusType = 'critical' | 'warning' | 'good' | 'unknown';
```

Comment:
Wondering about naming of 'good' vs. 'ok' -- have some inconsistencies within Grommet and this would be a good dev discussion topic to choose a direction we'd like to go long-term.